docs/testing/debug-add-ins-using-devtools-edge-legacy.md

```diff
@@ -57,12 +57,24 @@ To determine which browser you're using, see [Browsers used by Office Add-ins](.
 
 If your add-in uses the Office Dialog API, the dialog runs in a separate process from the task pane (if any) and the tools must attach to that process. Follow these steps.
 
-1. Run the add-in and the tools. 
+1. Run the add-in and the tools.
 1. Open the dialog and then select the **Refresh** button in the tools. The dialog process is shown. Its name comes from the `<title>` element in the HTML file that is open in the dialog.
 1. Select the process to open it and debug just as described in the section [Debug a task pane add-in using Microsoft Edge DevTools Preview](#debug-a-task-pane-add-in-using-microsoft-edge-devtools-preview).
 
    :::image type="content" source="../images/edge-devtools-with-add-in-and-dialog-processes.png" alt-text="Screenshot of Edge DevTools showing a process named My Dialog.":::
 
-## Install a version of Office that uses Microsoft Edge Legacy
+## Switch to the Edge Legacy webview
 
```

Comment:
The content in this section seems to be the same (or pretty similar) to that in the other files. Any reason not to use an include?

Reply:
It is using includes. But the headings and opening paragraph need to be different. One article is about Edge Legacy and others are about IE.